### ISSUE TITLE
Memorial Episode Toggled by 1st Episode

### ISSUE THREAD

**Issue by violet-r** [MagiCircles/BanGDream#27]
#If possible, memorial episode should be only able to be “ticked” when 1st episode is also chosen- you must unlock 1st episode in order to view memorial episode. #

**Comment by LatidoReMe** [comment]
That's not true though. You can unlock memorial before 1st if they're full leveled (albeit it is silly to do so, but still) so it might be better to make it "if max leveled, then toggle memorial episode".
